README.md

```diff
@@ -1,27 +1,32 @@
 # NServiceBus.RavenDB
 
-The official [NServiceBus](https://github.com/Particular/NServiceBus) persistence implementation for [RavenDB.](https://ravendb.net/)
+NServiceBus.RavenDB is the official NServiceBus persistence implementation for [RavenDB](https://ravendb.net/).
 
-Learn more about NServiceBus.RavenDB through our [documentation.](http://docs.particular.net/nservicebus/ravendb/)
+It is part of the [Particular Service Platform](https://particular.net/service-platform), which includes [NServiceBus](https://particular.net/nservicebus) and tools to build, monitor, and debug distributed systems.
 
-If you are interested in contributing, please follow the instructions [here.](https://github.com/Particular/NServiceBus/blob/develop/CONTRIBUTING.md)
+See the [RavenDB Persistence documentation](http://docs.particular.net/nservicebus/ravendb/) for more details on how to use it.
 
-## Running the tests
+## Running tests locally
 
-Running the tests requires RavenDB 5.2 and two environment variables. One named `CommaSeparatedRavenClusterUrls` containing the URLs, separated by commas, to connect to a RavenDB cluster to run cluster-wide transaction tests. The second one named `RavenSingleNodeUrl` containing the URL of a single node RavenDB instance to run non-cluster-wide tests. The tests can be run with RavenDB servers hosted on a Docker container.
+Running the tests requires RavenDB 5.2 and two environment variables: 
+
+1. `CommaSeparatedRavenClusterUrls` containing the URLs, separated by commas, to connect to a RavenDB cluster to run cluster-wide transaction tests
+1. `RavenSingleNodeUrl` containing the URL of a single node RavenDB instance to run non-cluster-wide tests
+
+The tests can be run with RavenDB servers hosted on a Docker container.
 
 ## CI Workflow
 
 The [CI workflow](/.github/workflows/ci.yml) requires two secrets unique to RavenDB to be defined both as Actions and Dependabot secrets:
 
-* RAVENDB_LICENSE: A RavenDB development license, expressed as JSON, but all on one line, with escaped quotes `\"`
-* PASSPHRASE: Any phrase used to encrypt the connection information between steps in the CI
+1. RAVENDB_LICENSE: A RavenDB development license, expressed as JSON, but all on one line, with escaped quotes `\"`
+1. PASSPHRASE: Any phrase used to encrypt the connection information between steps in the CI
```

Comment:
Any phrase used to encrypt the connection information between steps in the CI
- How do we do this in our current ci.yml?

Reply:
Good catch, looks like it's obsolete, I'll remove it.